### DISCUSSION TITLE
CPU temp on app behaves like it's hard locked

### DISCUSSION THREAD

**Discussion by Ludevic2000** [seerge/g-helper#734] [Q&A]
Hey, thanks for the amazing work and the opportunity to finally get rid of AC. 

I have been using G-Helper for a month, with no bugs encountered whatsoever.

For like a day or two however, G-Helper behaves like the CPU temp is locked on 45°C, and monitors the fan speed accordingly. Of course the cpu temp varies whenever i'm gaming etc. , but the fact that the software thinks the temperature is 45 degrees, prevents it from correctly managing the speed of the fans, and forces me to manually manage the latter by creating a speed plateau whatever the temperature, if not the it easily reaches 100 degrees.

All my drivers are up to date, fan speed controls are ok, the program behaves perfectly otherwise, undervolting works fine, and i use MSI afterburner to monitor the accurate temperatures, among other infos.

I use an Asus G733 with a ryzen 5900hx and rtx 3080.

I can't wrap my head around what's happening here, if you have an idea, i thank you in advance for sharing it.

Some illustrations : 

![image](https://github.com/seerge/g-helper/assets/138794429/d8a061a3-8397-4a66-b97a-ac20fbbe6bdd)

<img width="818" alt="screenshot 2023-07-06 162134" src="https://github.com/seerge/g-helper/assets/138794429/81d59a33-5186-4e1a-a1e3-0e21c3fb4c27">

**Comment by seerge** [comment]
@Ludevic2000 hello, app doesn't manage your fans actually :) your BIOS does based on it's own temperature sensor(s). G-helper just shows same sensor data.

1. May be it's some hiccup with bios/sensor ? does temperature ever drop below 45? 
2. What do apps like HWInfo show (they have a lot of ways to get readings)?
3. Try to reboot or even [hardware reset](https://github.com/seerge/g-helper#how-do-i-do-a-hardware-reset-on-a-laptop) mb it will wake up your temperature sensor :)

**Reply by Ludevic2000** [reply]
oh ok, i will try the harware reset. Unfortunately HWInfo does not show much, apart from the fact that it gave a similar and reasonable temperature reading (the temperature increasing when the CPU is under load). 

Thank you for the tip !
